### PR TITLE
Add basic scaffold for payment gateways

### PR DESCRIPTION
**Pull request by varunvasudevan94**
Added minimal support for payment gateway (its a test link so won't actually debit/credit any amounts)
Easy to toggle into production mode

Attaching a GIF for showcase
payments_razorpay

There is an option in razorpay to set redirection (we can set it to some thank you page)
Post payments a confirmation mail is also sent (email field is made mandatory)
image

ref - https://github.com/shivikawedsyash/invitation/issues/3
TODO - store the receipt in some form from our application

### CODE REVIEW

**Review by ghardikg99 — APPROVED** [review]
looks good

**Comment by shivikawedsyash** [comment]
ye startup bna le fir?

File: index.html [hunk 1]
```diff
@@ -70,6 +71,9 @@ <h1>Yash</h1>
 <!-- Countdown Timer -->
 <div id="time" style="text-align: center; margin-top: 20px;"></div>
 
+<div id="wedding" class="wrap">
+   <button class="gift-btn" id="gift-btn">Gift Now&nbsp;&nbsp;&nbsp;</button>
```
Comment (shivikawedsyash) [inline index.html:75]: the style element seems to be missing

Comment (varunvasudevan94) [reply index.html:75]: raised this PR; https://github.com/shivikawedsyash/invitation/pull/12

**Pull request merged**